### DISCUSSION TITLE
Upgrade Wikijs 2.x to latest version on Heroku

### DISCUSSION THREAD

**Discussion by DominikHoffendahl** [requarks/wiki#3367] [Help / Questions]
I already saw this issue related to Wikijs 1.x and was wondering if there is a simple way to upgrade your Wikijs deployment on Heroku für version 2.x, without creating a new Heroku App.

Is it enough to trigger a re-deploy? It depends on the version taken from the [dockerfile](https://github.com/Requarks/wiki-heroku/blob/2.x/Dockerfile) , right?

**Comment by DominikHoffendahl** [comment]
I didn't manage to ugprade yet by just re-deploying it. So my initial question is still valid

**Comment by crrmacarse** [comment]
Up with this

We've initialized our wiki via heroku. I can't seem to download the source code and change the dockerfile

https://help.heroku.com/FZDDCBLB/how-can-i-download-my-code-from-heroku
